source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3171,6 +3281,10 @@ export class ColumnSet extends CardElement {
             invokeSetParent(this._selectAction, this);
         }
     }
+
+    get columns(): Array<Column> {
```

Comment:
Needs to be removed. Developers are supposed to call addColumn that properly sets the parent.